docs/docs/kds-rest-api.md

```diff
@@ -2,21 +2,21 @@
 
 ## Introduction
 
-This page describes how applications can communicate with the Key Derivation Service, also known as KDS. The service is made accessible
-through its REST API endpoints which are presented below. As many applications, KDS also uses JSON to send and receive data.
+This page describes how applications can communicate with the Key Derivation Service (KDS). The KDS is a web server 
```

Comment:
I wouldn't mention "web server". 

It exposes an HTTP REST API for ... (something like that)

Reply:
Fixed in [this PR](https://github.com/R3Conclave/conclave-core-sdk/pull/50).